java/arcs/core/storage/ReferenceModeStore.kt

```diff
@@ -126,11 +128,27 @@ class ReferenceModeStore private constructor(
      */
     private val versions = mutableMapOf<ReferenceId, MutableMap<FieldName, Int>>()
 
+    val backingStore = BackingStore<CrdtData, CrdtOperation, Any?>(
```

Comment:
Should/could this be `private`?

Reply:
It could; I left it as-is for now as there are a number of tests referencing the property directly. I could change it and fix up the affected tests.

Reply:
The test are doing non-trivial things with the backing store, so I just added a `Visibility` annotation for now.